### ISSUE TITLE
Feature extraction

### ISSUE THREAD

**Issue by craffel** [craffel/pretty-midi#18]
Some enthusiastic individual might be interested in implementing all of the features described in section 4.5 here:

http://jmir.sourceforge.net/publications/PhD_Dissertation_2010.pdf

**Comment by bzamecnik** [comment]
It seems these metrics are already available in `music21` (web.mit.edu/music21/doc/moduleReference/moduleFeaturesJSymbolic.html).

**Comment by craffel** [comment]
Unfortunately (at least historically, maybe this isn't true any more?) parsing a MIDI file with music21 is very slow.  But it's cool to see a pure-Python implementation of the jSymbolic features, anyways.  This issue is pretty low priority.

**Comment by bzamecnik** [comment]
Yeah, that's true that music21 is really complex and slow. I just wanted to point out another Python implementation that might serve as an inspiration for porting that to pretty-midi.
